helm/values.yaml

```
@@ -1,6 +1,12 @@
 # imagePullPrefix: icr.io
 # imagePullSecret: ibm-entitlement-key
 
+# The following global values could be overridden by top-level values
+global:
+  tetheredNamespace: [tenant1,tenant2]
```

```suggestion
  tetheredNamespace: []
```
leave it empty by default